README.md

```diff
@@ -159,6 +159,7 @@ rs.stop_trace
 #### `Rotoscope#flatten(dest)`
 Reduces the output data to a list of method invocations and their caller, instead of all `call` and `return` events. Methods invoked at the top of the trace will have a caller entity of `<ROOT>` and a caller method name of `unknown`. `dest` is either a filename or an instance of IO, or IO-like, object.
 
+_Note: This form is significantly slower than passing `flatten: true` to the constructor of `Rotoscope` for large trace files due to differing implementations._
```

Comment:
What do you think about just having the `flatten` option, and removing this function?